form.yml

```diff
@@ -17,24 +17,24 @@ attributes:
     widget: select
     label: "Node type"
     help: |
-      - **any** - (*12 cores*) Use any available Oakley node. This reduces the
+      - **vis** - (*28 cores*) This node includes an NVIDIA Tesla P100 GPU
```

Comment:
Do we want `vis` node types to be the default? They will have to wait longer in queue.

Reply:
That was a requirement for the class. Not a requirement for the port for everyone to use. Now we have a separate class specific branch.

Reply:
Ok, I'll bump it back.